deployer/pipeline_deployer.py

```diff
@@ -139,6 +157,7 @@ def _create_pipeline_job(
         """  # noqa: E501
         job = aiplatform.PipelineJob(
             display_name=self.pipeline_name,
+            job_id=self.pipeline_display_name,
```

Comment:
Do you have an example of how these two arguments render in the vertex UI ?

Reply:
Do you mean this? :)

<img width="657" alt="image" src="https://github.com/artefactory/vertex-pipelines-deployer/assets/101176396/2031d371-7524-47f7-8074-c66aec667cf3">


![image](https://github.com/artefactory/vertex-pipelines-deployer/assets/101176396/22e3eb72-e080-4659-8d9b-ca30e66698ef)